test_connections.py

```diff
@@ -0,0 +1,12 @@
+import urllib.request
```

Comment:
this tests package does not seem to contain any actual tests

Reply:
ack

Reply:
the test packages will contain a series of boolean functions that will return true or false according to our needs, that way more elaborate tests can be performed if need be.